### PR TITLE
Add support for detection and addition of ome/ngff "standard" into assets summary

### PR DESCRIPTION
**Pull request by yarikoptic**
TODOs

- [x] test

we will need to release so that dandi-archive could take advantage of it

### CODE REVIEW

File: dandischema/models.py [hunk 1]
```diff
@@ -807,6 +807,12 @@ class StandardsType(BaseType):
     identifier="RRID:SCR_016124",
 ).model_dump(mode="json", exclude_none=True)
 
+ome_ngff_standard = StandardsType(
+    name="OME/NGFF Standard",
+    # Seems no dedicated RRID, only for OME itself?
+    # identifier="RRID:SCR_???",
```
Comment (satra) [inline dandischema/models.py:813]: @joshmoore - any thoughts here on identifier for the ome/ngff standard? or could we mint an RRID for it?

Comment (joshmoore) [reply dandischema/models.py:813]: :wave: @satra:

 * No identifier (I know of) in use at the moment.
 * Does it need to be an RRID?
 * Closest I know of is https://fairsharing.org/FAIRsharing.9af712 (https://doi.org/10.25504/FAIRsharing.9af712)

Comment (satra) [reply dandischema/models.py:813]: thanks @joshmoore - doesn't have to be RRID, we can use any persistent identifier. we can use the doi.

Comment (joshmoore) [reply dandischema/models.py:813]: Let me run it by a few folks Monday AM, and I'll get back to you.

Comment (joshmoore) [reply dandischema/models.py:813]: Sorry for the wait, @satra. No other identifiers from our side. If the DOI works for the moment, then :+1: (How hard would this be to change in the future?)

Comment (satra) [reply dandischema/models.py:813]: hyphen is fine. we can change it in the future. will just need to encode the change and the mapping.

Comment (joshmoore) [reply dandischema/models.py:813]: Sounds good.

Comment (yarikoptic) [reply dandischema/models.py]: Thank you gentleman, I added DOI identifier.

File: dandischema/models.py [hunk 2]
```diff
@@ -807,6 +807,11 @@ class StandardsType(BaseType):
     identifier="RRID:SCR_016124",
 ).model_dump(mode="json", exclude_none=True)
 
+ome_ngff_standard = StandardsType(
+    name="OME/NGFF Standard",
```
Comment (satra) [inline dandischema/models.py:811]:
```suggestion
    name="OME-NGFF Standard",
```

Comment (satra) [reply dandischema/models.py:811]: switched to @joshmoore request for hyphen

**Comment by yarikoptic** [comment]
cli fail unrelated
```
=========================== short test summary info ============================
FAILED tests/test_dandiarchive.py::test_parse_api_url[https://gui.dandiarchive.org/#/dandiset/000001-parsed_url0] - requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
= 1 failed, 706 passed, 1 skipped, 7 xfailed, 9 xpassed, 26 warnings in 720.33s (0:12:00) =
Error: Process completed with exit code 1.
```

Let's proceed.

**Pull request merged**